prosemirror-jdita/src/schema.ts

```diff
@@ -127,18 +200,44 @@ export function getDomAttr(nodeName: string, attr: string): string {
     : 'data-j-' + attr;
 }
 
+/**
+ * Create default node attributes
+ *
+ * @param attrs - The attributes of the node
+ * @returns A map of the attributes with default values
+ */
 export function defaultNodeAttrs(attrs: string[]): any {
   return attrs.reduce((result, field) => {
     result[field] = { default: '' };
     return result;
   }, {} as Record<string, { default: string }>);
 }
 
-function defaultTravel(node: typeof BaseNode, parent: typeof BaseNode, next: (nodeName: string, parent: typeof BaseNode) => void): NodeSpec {
+/**
+ * Travel the node and generate the node spec
+ *
+ * @remarks
+ * NodeSpec is a description of a node type, used when defining a schema.
+ * @see {@link https://prosemirror.net/docs/ref/#model.NodeSpec} for more info
+ *
+ * @param nodeName - The name of the node
+ * @param parent - The parent of the node
+ * @param next - Next travel function
+ * @returns NodeSpec
+ */
+function defaultTravel(
+  node: typeof BaseNode,
+  parent: typeof BaseNode,
+  next: (nodeName: string, parent: typeof BaseNode) => void): NodeSpec {
+  // get the children of the node
   const children = (SCHEMA_CHILDREN[node.nodeName] || getChildren)(node.childTypes);
+  // make the distinction between a node and a mark
   const isNode = IS_MARK.indexOf(node.nodeName) < 0;
+  // get the content and group of the node
   const [content, group] = isNode ? SCHEMA_CONTENT[node.nodeName] : [undefined, undefined];
+  // get the attributes of the node
   const attrs = (NODE_ATTRS[node.nodeName] || defaultNodeAttrs)(['parent', ...node.fields]);
+  // create the node spec
```

Comment:
Redundant comment. The code's intention is clear!

Reply:
Fixed with 1b1a07a.